var/spack/repos/builtin/packages/espanso/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Espanso(Package):
+    """Cross-platform Text Expander written in Rust"""
+
+    homepage = "https://github.com/federico-terzi/espanso"
+    url      = "https://github.com/federico-terzi/espanso/releases/download/v0.6.3/espanso-linux.tar.gz"
+
+    maintainers = ['federico-terzi']
```

Comment:
@frederico-terzi are you okay with being listed as an official maintainer of this package? You don't have to be a Spack expert, it just gives us someone to ping in case a user reports build issues or wants to modify the package.

Reply:
The ping was misspelled: @federico-terzi ^^^

Reply:
Hey,
Unfortunately, I'm not practical of spack and I already have a lot of things to maintain at the moment :( 
That said, I'm definitely not against publishing it on this package manager. @zicklag would you be interested in becoming the official maintainer yourself? If you experience some issue you can definitely ping me, but I have quite a lot going on at the moment so I'd rather avoid this one...

Reply:
I'm not sure how much time I'd get for it, but it won't hurt to put me on it. I'll do that.